packages/aws-cdk/package.json

```diff
@@ -65,7 +65,8 @@
     "sinon": "^9.2.4",
     "ts-jest": "^27.1.3",
     "ts-mock-imports": "^1.3.8",
-    "xml-js": "^1.6.11"
+    "xml-js": "^1.6.11",
+    "axios": "^0.25.0"
```

Comment:
This needs to go somewhere into some test runner script where we also install `jest`, not here.

Reply:
`yarn build` fails if I removed `axios` from `package.json` as it is required to invoke the exposed api to validate the integration with sam.